components/TextFields/src/MDCTextInputBorderView.m

```diff
@@ -0,0 +1,171 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCTextInputBorderView.h"
+
+static NSString *const MDCTextInputBorderViewBorderFillColorKey =
+    @"MDCTextInputBorderViewBorderFillColorKey";
+static NSString *const MDCTextInputBorderViewBorderPathKey =
+    @"MDCTextInputBorderViewBorderPathKey";
+static NSString *const MDCTextInputBorderViewBorderStrokeColorKey =
+    @"MDCTextInputBorderViewBorderStrokeColorKey";
+
+static inline NSString * _Nullable MDCNSStringFromCGLineCap(CGLineCap lineCap) {
```

Comment:
naming-nit: You typically don't include the object prefixes in the function name.
Consider [UIView initWithCoder:].  It is initWithCoder, not initWithNSCoder.
Or NSClassFromString, not NSClassFromNSString.
s/MDCNSStringFromCGLineCap/MDCStringFromCGLineCap/
However, CGLineCap is in CoreGraphics, which is still dealing with CGRefs instead of object pointers, so it still has the prefix.
Maybe,...  naming is hard.